backend/templates/v2/ngo-account/redirections/main.html

```diff
@@ -13,6 +13,10 @@
   {% endblocktrans %}
 {% endblock %}
 
+{% block additional_headers %}
+  <script src="https://unpkg.com/htmx.org@2.0.4" integrity="sha384-HGfztofotfshcF7+8n44JQL2oJmowVChPTg48S+jvZoztPfvwD79OC/LTtG6dMp+" crossorigin="anonymous"></script>
+{% endblock %}
```

Comment:
I'd rather we didn't have to bring in a new library (yes, albeit a small one) only for one page.
We're using AlpineJS for the interactive functionality in the website. If you want to work on this, we're most likely not going to approve code that brings in a dependency that has the same use-case as a different one we already have, [so please consider using AlpineJS](https://alpinejs.dev/directives/model).